core/dplug/core/ringbuf.d

```diff
@@ -138,6 +138,22 @@ struct RingBufferNoGC(T)
             return _data.ptr[(_first + index) % _data.length];
         }
 
+        /// Sets the item with the corresponding index in the queue to the passed value.
+        ///
+        /// Caution: Circumvents the purpose of a buffer.
+        void opIndexAssign(T value, size_t index)
```

Comment:
What do you think about it? (I'd guess, using this will usually make no sense :see_no_evil: But maybe it's useful to someone...)

Reply:
If you don't use it, then it shoudln't exist.

Reply:
> maybe it's useful to someone...

Dplug is precisely _not_ written with this in mind :)